docs/Validation-and-Serialization.md

```diff
@@ -198,6 +198,62 @@ fastify.post('/the/url', { schema }, handler)
 
 *Note that Ajv will try to [coerce](https://github.com/epoberezkin/ajv#coercing-data-types) the values to the types specified in your schema `type` keywords, both to pass the validation and to use the correctly typed data afterwards.*
 
+The Ajv default configuration in Fastify doesn't support coercing array parameters in querystring. However, Fastify allows [`customOptions`](Server.md#ajv) in Ajv instance. The `coerceTypes: 'array'` will coerce one parameter to a single element in array. Example:
+
+```js
+const opts = {
+  schema: {
+    querystring: {
+      type: 'object',
+      properties: {
+        ids: {
+          type: 'array',
+        },
+      },
+      required: ['ids']
```

Comment:
I think this should actually not use `required: ['ids']` here, otherwise it will not be possible to pass in an empty list of ids.

In order to support empty arrays, I think a default is needed instead:

```js
const opts = {
  schema: {
    querystring: {
      type: 'object',
      properties: {
        ids: {
          type: 'array',
          default: [],
        },
      },
    }
  }
}

```